.github/workflows/rust.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [ "main" ]
   pull_request:
-    branches: [ "main" ]
+    branches: "**"
```

Comment:
Wouldn't it be a good idea to have the check on all branches even on push

Reply:
I guess, I just don't think it makes sense to run the CI on branches with no pull request :thinking:

Reply:
I'd argue it could be used while developing to ensure that your code is (relatively) "clean" all the time, but I totally see the point

Reply:
I mean that is what a draft pr is for, but I'm willing to budge.

Reply:
Fixed